test/docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ruby:2.3.8
```

Comment:
I think you checked these in accidentally :)

Reply:
Ah, no thanks, please don't include changes that weren't discussed. :) Unless you're on windows, it's pretty easy to install bats. So I don't think we should add scripts and dockerfiles almost as long as the tests to run the tests.

Reply:
Fair enough. Pruned :)